src/modules/dashboards/utils/createCodeSnippet.ts

```diff
@@ -0,0 +1,72 @@
+type Snippet = 'body' | 'head';
+
+const createCodeSnippet = ({
+  projectId,
+  masterKey,
+  dashboardId,
+  type,
+}: {
+  projectId: string;
+  masterKey: string;
+  dashboardId: string;
+  type?: Snippet;
+}) => {
+  if (type === 'head') {
+    return `<script crossorigin src="https://cdn.jsdelivr.net/npm/keen-dashboard-builder@latest/dist/viewer.min.js"></script>`;
+  }
+  if (type === 'body') {
+    return `
+    <div id="root"></div>
+    <script type="text/javascript">
+       new KeenPublicDashboard({
+         container: '#root',
+         dashboardId: ${dashboardId},
+         backend: {
+           analyticsApiUrl: 'staging-api.keen.io',
+           dashboardsApiUrl: 'https://blob-service.us-west-2.test.aws.keen.io',
+         },
+         project: {
+           id: ${projectId},
+           masterKey: ${masterKey} 
+         },
+       }).render();
+      </script>
+    `;
+  }
+  return `
+  <!doctype html>
+  <html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=11" />
+    <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
The title should be different than WebPack plugin variable. Please, change it to for example `Public Dashboard`.